plugins/inputs/cloudwatch/cloudwatch_test.go

```diff
@@ -189,11 +209,11 @@ func TestSelectMetrics(t *testing.T) {
 	duration, _ := time.ParseDuration("1m")
 	internalDuration := config.Duration(duration)
 	c := &CloudWatch{
-		Region:    "us-east-1",
-		Namespace: "AWS/ELB",
-		Delay:     internalDuration,
-		Period:    internalDuration,
-		RateLimit: 200,
+		Region:     "us-east-1",
+		Namespaces: []string{"AWS/ELB"},
```

Comment:
Could we add a test case that defined a Namespace and Namespaces? Make sure they get added together.